app/uk/gov/hmrc/fileupload/write/envelope/Envelope.scala

```diff
@@ -23,19 +23,22 @@ import uk.gov.hmrc.fileupload.{FileId, FileRefId}
 
 object Envelope extends Handler[EnvelopeCommand, Envelope] {
 
+  val defaultMaxNumFilesCapacity = 100
+  val defaultMaxSize = 25 //MB
+
```

Comment:
Maybe rename the val `defaultMaxSizeInMb` removing the need for the comment and making it obvious wherever it is referenced.